bot/exts/evergreen/minesweeper.py

```diff
@@ -47,22 +47,19 @@ class Game:
     activated_on_server: bool
 
 
-GamesDict = typing.Dict[int, Game]
-
-
 class Minesweeper(commands.Cog):
     """Play a game of Minesweeper."""
 
     def __init__(self) -> None:
-        self.games: GamesDict = {}  # Store the currently running games
```

Comment:
Why comment was removed?

Reply:
I think it should be fairly obvious to what this attribute is, it describes itself. The comment has been deemed unnecessary.